src/components/Disclaimer/Disclaimer.tsx

```diff
@@ -4,14 +4,18 @@ import { DisclaimerWrapper, DisclaimerBody } from './Disclaimer.style';
 import LightTooltip from 'components/LightTooltip/LightTooltip';
 import { useModelLastUpdatedDate } from 'common/utils/model';
 import { Metric } from 'common/metric';
+import { CASE_GROWTH_DISCLAIMER } from 'common/metrics/case_growth';
+import { POSITIVE_RATE_DISCLAIMER } from 'common/metrics/positive_rate';
+import { HOSPITALIZATIONS_DISCLAIMER } from 'common/metrics/hospitalizations';
+import { CONTACT_TRACING_DISCLAIMER } from 'common/metrics/contact_tracing';
+const Disclaimer = ({ metricName }: { metricName: number }) => {
+  const METRIC_TO_DISCLAIMER: { [metricName: number]: string } = {
+    [Metric.CASE_GROWTH_RATE]: CASE_GROWTH_DISCLAIMER,
+    [Metric.POSITIVE_TESTS]: POSITIVE_RATE_DISCLAIMER,
+    [Metric.HOSPITAL_USAGE]: HOSPITALIZATIONS_DISCLAIMER,
+    [Metric.CONTACT_TRACING]: CONTACT_TRACING_DISCLAIMER,
+  };
```

Comment:
I'd recommend putting this METRIC_TO_DISCLAIMER map in common/metric.ts so it's more centralized and we're more likely to remember to update it when we add / change metrics.

Reply:
moved !